tools/mo/openvino/tools/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -410,12 +410,12 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
             if op.has_and_set('rt_info'):
                 op.out_node(0)['rt_info'] = op.rt_info
 
-        restore_tensor_names(op)
-
         # operations postprocessing with some special types
         if op.soft_get('type') in postprocessing_op_nodes:
             postprocessing_op_nodes[op.type](op)
 
+        restore_tensor_names(op)
+
```

Comment:
can you explain please how tensor name can appear for output port previously being disconnected?

Reply:
It was disconnected but it is exist and has name in IR, but we cannot restore this name because inside MO we use edge attributes to store it. So we need to add fake output before. 
![image](https://user-images.githubusercontent.com/56019755/151380380-594a8339-78de-4453-a6a0-db3847e75e48.png)